### DISCUSSION TITLE
Get API

### DISCUSSION THREAD

**Discussion by Coupeswiss** [FlowiseAI/Flowise#709] [General]
does anyone know how to use the GET api feature ? I would like to call google places API within flowise

**Comment by chungyau97** [comment]
Hi @Coupeswiss,

**Request Get**
URL: Agent will make call to this exact URL. If not specified, agent will try to figure out itself from AIPlugin if provided.
Description: Acts like a prompt to tell agent when it should use this tool
Headers:  Extra source of information for each API call you make

I recommend you using [Custom Tool](https://docs.flowiseai.com/tools/custom-tool) and there are a lot of example shown below:
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/df14b69d-9d95-4951-a720-0d0b78e50b41)
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/605f2228-81e2-494c-9c6a-c463931ef5a8)
